scripts/tank_cmd.py

```diff
@@ -189,19 +197,36 @@ def show_help(log):
         log.info(x)
 
 
-def ensure_authenticated():
+def ensure_authenticated(cmd_line_credentials):
     """
     Make sure that there is a current toolkit user set.
     May prompt for a login/password if needed
+
+    :param cmd_line_credentials: Dictionary of credentials passed from the
```

Comment:
what's in this dictionary? List of keys would be great or even better some examples!